vm-creation.md

```diff
@@ -38,3 +38,84 @@ spec:
         lun: 2
         targetPortal: iscsi-demo-target.kube-system.svc.cluster.local
 ```
+
+### Expose VirtualMachine as a service
+
+Once the VirtualMachine is started, in order to connect to a VirtualMachine, you can create `Service` object for a VirtualMachine.
+
+### Expose VirtualMachine as a NodePort service
+
+Expose the SSH port (TCP port 22) of a VirtualMachine running on KubeVirt.
+Here is an example of a NodePort service:
+
+```
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    kubevirt.io: virt-launcher
+    kubevirt.io/domain: testvm-ephemeral
+  name: testnodeport
+spec:
+  externalTrafficPolicy: Cluster
+  ports:
+  - name: nodeport
+    nodePort: 30000
+    port: 27017
+    protocol: TCP
+    targetPort: 22
+  selector:
+    kubevirt.io: virt-launcher
+    kubevirt.io/domain: testvm-ephemeral
+  sessionAffinity: None
+  type: NodePort
+status:
+  loadBalancer: {}
+```
+
+You just need to create `NodePort service` by using `kubectl`:
+
+```bash
+$ kubectl -f nodeport.yaml
+# OR
+$ kubectl get pod
+NAME                                   READY     STATUS    RESTARTS   AGE
+virt-launcher-testvm-ephemeral-9bqv4   2/2       Running   0          10m
+$ kubectl expose pod virt-launcher-testvm-ephemeral-mxjh8 --port=27017 --target-port=22 --type=NodePort --name=nodeportsvc
+```
+
+Service	can be listed by querying for the service objects:
+
+```bash
+$ kubectl get service
+NAME           TYPE       CLUSTER-IP      EXTERNAL-IP   PORT(S)           AGE
+nodeportsvc    NodePort   172.30.232.73   <none>        27017:30000/TCP   5m
+```
+
+Connect the VirtualMachine by node IP and node port outside the clusternetwork:
```

Comment:
Connect **to** the VirtualMachine by **using a** node IP and the node port from outside of the cluster-network: